### ISSUE TITLE
WindowBuilder development plugin should be installed as part of Oomph setup

### ISSUE THREAD

**Issue by ptziegler** [eclipse-windowbuilder/windowbuilder#644]
The plugin [org.eclipse.wb.dev](https://github.com/eclipse-windowbuilder/windowbuilder/tree/master/org.eclipse.wb.dev) contains a custom builder and validator for the WindowBuilder xml files.

We should try to  re-enable this validation and have the plug-in be installed as part of the setup.
Right now, the only thing it does is causing the following warning:
> Unknown referenced nature: org.eclipse.wb.dev.wbNature

I assume the entry point for the Oomph script is the P2 Director, where we would have to add it as a new... project?
I don't see how the tool would be able to install the plug-in directly from the workspace, so I also believe that we need to publish it to our update site, together with the other artifacts, and then consume it from there...

In short, I need to read up on how Oomph actually works :sweat_smile:

**Comment by merks** [comment]
Yes we should publish it to the update site.  The rest is trivial.

**Comment by ptziegler** [comment]
Fixed with #651. Both the WindowBuilder Dev plugin as well as SpotBugs are installed during the Oomph setup.
